src/cpp/include/openvino/genai/llm_pipeline.hpp

```diff
@@ -82,6 +82,8 @@ class DecodedResults {
     }
 };
 
+class LLMPipelineImplBase;
```

Comment:
please, move NPU impl to a dedicated files so that common refactoring should avoid changing NPU specifics

Reply:
Done